### DISCUSSION TITLE
react-native example ?

### DISCUSSION THREAD

**Discussion by xseignard** [NiGhTTraX/ts-monorepo#316] [Ideas]
Hello,

Thanks for your nice work. It would be very interesting to have an example with react-native too.

Thanks for your blog post and maintaining this repo 👍

**Comment by NiGhTTraX** [comment]
Hey @xseignard. Unfortunately I don't have any React Native experience so I don't know what implications a monorepo has on the bundler used there. If anyone wants to contribute an example I'd be happy to review it and help getting it merged.
